src/Makefile.am

```diff
@@ -82,7 +82,8 @@ AM_CPPFLAGS += \
 forked_daapd_LDADD = \
 	$(FORKED_LIBS) \
 	$(FORKED_OPTS_LIBS) \
-	$(COMMON_LIBS)
+	$(COMMON_LIBS) \
+	-lm
```

Comment:
Can't hardcode this here. If anything it should go in configure.ac. But as mentioned below you could also probably do without it.